Home_Page.py

```diff
@@ -0,0 +1,33 @@
+from selenium import webdriver
```

Comment:
Put this file in a directory named `coinbase/`. Rename the file to be `homepage.py`. Going to try to stick to PEP8 basics. We should stick to this for all filenames. Consider changing any you see that don't comply when you're working on a related ticket.

https://www.python.org/dev/peps/pep-0008/#package-and-module-names

The name of the file should be the url extension when it makes sense (in the case of coinbase, "signin," "dashboard," etc.).

imports should read `import pages.coinbase.homepage`, `pages.coinbase.signin`. Other sites should read, e.g., `pages.wikipedia.homepage`, `pages.wikipedia.wiki`. etc. I'll add this to the upcoming confluence page.